tools/ci/download_bin.sh

```diff
@@ -1,52 +1,63 @@
 #!/bin/bash
 
-org=FISCO-BCOS
-repo=FISCO-BCOS
-branch=master
+project=FISCO-BCOS/FISCO-BCOS
+cdn_link_header="https://www.fisco.com.cn/cdn/fisco-bcos/releases/download"
+download_branch=
 output_dir=bin/
-download_gm=
 download_mini=
+download_version=
+latest_version=
+download_timeout=240
 
 help() {
-    echo $1
-    cat << EOF
+    echo "
 Usage:
-    -b <Branch>                   Default master
-    -o <Output Dir>               Default ./
+    -v <Version>           Download binary of spectfic version, default latest
+    -b <Branch>            Download binary of spectfic branch
+    -o <Output Dir>        Default ./bin
+    -l                     List released FISCO-BCOS versions
+    -m                     Download mini binary, only works with -b option
     -h Help
 e.g
-    $0 -b master
-EOF
+    $0 -v 2.6.0
+"
 
 exit 0
 }
 
-parse_params()
+LOG_INFO()
 {
-while getopts "b:o:hm" option;do
+    local content=${1}
+    echo -e "\033[32m[INFO] ${content}\033[0m"
+}
+
+parse_params(){
+while getopts "b:o:v:hml" option;do
     case $option in
     o) output_dir=$OPTARG;;
-    b) branch="$OPTARG";;
+    b) download_branch="$OPTARG";;
+    v) download_version="${OPTARG//[vV]/}";;
     m) download_mini="true";;
+    l) LOG_INFO "Show the history releases of FISCO-BCOS " && curl -sS https://gitee.com/api/v5/repos/${project}/tags | grep -oe "\"name\":\"v[2-9]*\.[0-9]*\.[0-9]*\"" | cut -d \" -f 4 | sort -V && exit 0;;
     h) help;;
     *) help;;
     esac
 done
 }
 
-download_artifact_linux()
-{
-    build_num=$(curl https://circleci.com/api/v1.1/project/github/${org}/${repo}/tree/${branch}\?circle-token\=\&limit\=1\&offset\=0\&filter\=successful 2>/dev/null| grep build_num | sed "s/ //g"| cut -d ":" -f 2| sed "s/,//g" | sort -u | tail -n1)
+download_artifact_linux(){
+    local branch="$1"
+    build_num=$(curl https://circleci.com/api/v1.1/project/github/${project}/tree/${branch}\?circle-token\=\&limit\=1\&offset\=0\&filter\=successful 2>/dev/null| grep build_num | sed "s/ //g"| cut -d ":" -f 2| sed "s/,//g" | sort -u | tail -n1)
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5987732)